eo-maven-plugin/src/main/java/org/eolang/maven/Home.java

```diff
@@ -0,0 +1,257 @@
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import com.jcabi.log.Logger;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.cactoos.Input;
+import org.cactoos.Text;
+import org.cactoos.io.InputOf;
+import org.cactoos.io.OutputTo;
+import org.cactoos.io.TeeInput;
+import org.cactoos.scalar.IoChecked;
+import org.cactoos.scalar.LengthOf;
+
+/**
+ * Class for saving and loading files.
+ *
+ * @since 0.27
+ * @todo #1105:30min create load function (it has to be able read by path)
+ *  and add new unit test for added load function
+ */
+@SuppressWarnings("PMD.TooManyMethods")
+public class Home {
+    /**
+     * Current working directory.
+     */
+    private final Path cwd;
+
+    /**
+     * Ctor.
+     */
+    Home() {
+        this(Paths.get(""));
+    }
+
+    /**
+     * Ctor.
+     * @param path Path
+     * @todo #1105:30min we need to modify function. It has to change
+     *  path by replacing bad characters (not utf-8) or maybe
+     *  we can invent any other solution
+     */
+    Home(final Path path) {
+        this.cwd = path;
+    }
+
+    /**
+     * Saving string.
+     * @param path Path
+     * @param str String
+     * @return Saved bytes
+     * @throws IOException If fails
+     */
+    public long save(final Path path, final String str) throws IOException {
+        return this.write(path, new InputOf(str));
+    }
+
+    /**
+     * Saving text.
+     * @param path Path
+     * @param txt Text
+     * @return Saved bytes
+     * @throws IOException If fails
+     */
+    public long save(final Path path, final Text txt) throws IOException {
+        return this.write(path, new InputOf(txt));
+    }
+
+    /**
+     * Saving stream.
+     * @param path Path
+     * @param stream Input stream
+     * @return Saved bytes
+     * @throws IOException If fails
+     */
+    public long save(final Path path, final InputStream stream) throws IOException {
+        return this.write(path, new InputOf(stream));
```

Comment:
@yasamprom you have a lot of code duplication in these methods. It's better to use existing methods, instead of calling `this.write` from each one. See how it's done in ctors of `Save`

Reply:
> @yasamprom you have a lot of code duplication in these methods. It's better to use existing methods, instead of calling `this.write` from each one. See how it's done in ctors of `Save`

@yegor256 So, we will use class like that: `new Home().Save(path, data).write();` where `Save` inits class fields `Path path` and `Input input`
Did I get the idea?

Reply:
@yasamprom no, this is wrong:

```
a(x) {
  f(Y(x));
}
a(y) {
  f(y);
}
```

this is right:

```
a(x) {
  a(Y(x));
}
a(y) {
  f(y);
}
```